recipe/meta.yaml

```
@@ -3,12 +3,14 @@ package:
   version: 1.0
 
 build:
-  number: 1
+  number: 2
 
 test:
   commands:
     - test "$PYTHONNOUSERSITE" == "1"         # [unix]
     - test "$R_LIBS_USER" == "-"              # [unix]
+    - test -f "$PREFIX/etc/conda/activate.d/user-package-isolation.fish"   # [unix]
+    - test -f "$PREFIX/etc/conda/deactivate.d/user-package-isolation.fish" # [unix]
```

The linter requires a minimum of two spaces. The suggested edit below adds a space to both lines to keep the jinja2 preprocessors aligned. Please accept the edit or push your own commit to fix this.

```suggestion
    - test -f "$PREFIX/etc/conda/activate.d/user-package-isolation.fish"    # [unix]
    - test -f "$PREFIX/etc/conda/deactivate.d/user-package-isolation.fish"  # [unix]
```